docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -212,6 +212,12 @@ You can also add a test for the deprecation warning::
             # invoke deprecated behavior
             ...
 
+You should add a ``RemovedInDjangoXXWarning`` comment above code which does not
```

Comment:
In order to reduce the duplication of "you should also add" and "you should add", I would propose the following instead:

`It's important to include a ``RemovedInDjangoXXWarning`` comment above code which does not ...`

And I would recommend adding a small example, this way it's more likely that future contributors would get the diff right the first time they try.

Reply:
I found it difficult thinking of a good example, but I gave it a go :+1: